src/ngraph/op/topk.hpp

```diff
@@ -161,8 +161,13 @@ namespace ngraph
                     copy_with_new_args(const NodeVector& new_args) const override;
 
                 virtual size_t get_version() const override { return 1; }
-                size_t get_axis() const { return m_axis; }
-                void set_axis(const size_t axis) { m_axis = axis; }
+                size_t get_axis() const { return m_normalized_axis; }
+                int64_t get_origin_axis() const { return m_axis; }
+                void set_axis(const size_t axis)
+                {
+                    m_normalized_axis = axis;
+                    m_axis = axis;
+                }
```

Comment:
We should consider change `set_axis` to accept `int64_t` instead of `size_t` here.

Reply:
Replace the `size_t`s with `int64_t`.
size_t is very platform-specific.

Reply:
If changing to `int64_t` call `normalize_axis()`.

Reply:
Done, changed to `int64_t`. setter call normalization too.
It will be refactored after https://github.com/NervanaSystems/ngraph/pull/4299 merge.